source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,17 @@
+//
+// View Selector component
+//
+
+.pf-view-selector__btn-link {
```

Comment:
I did it this way because I think it's fundamental to BEM - no nesting except for pseudo-classes. Otherwise, I think it's inevitably going to turn into specificity struggles and cause difficulties when trying to reuse elements in different places.